packages/block-library/src/quote/deprecated.js

```diff
@@ -122,6 +116,47 @@ const deprecated = [
 			);
 		},
 	},
+	{
+		attributes: {
+			...blockAttributes,
+			value: {
+				type: 'string',
+				source: 'html',
+				selector: 'blockquote',
+				multiline: 'p',
+				default: '',
+			},
+		},
+		migrate( attributes ) {
+			// The old value attribute for quotes can contain:
+			// - a single paragraph: "<p>single paragraph</p>"
+			// - multiple paragraphs: "<p>first paragraph</p><p>second paragraph</p>"
+			const innerBlocks = parseWithAttributeSchema( attributes.value, {
+				type: 'array',
+				source: 'query',
+				selector: 'p',
+				query: {
+					value: {
+						type: 'string',
+						source: 'text',
+					},
+				},
+			} ).map( ( { value } ) =>
+				createBlock( 'core/paragraph', { content: value } )
+			);
+			return [ omit( attributes, [ 'value' ] ), innerBlocks ];
```

Comment:
Didn't `citation` change to `attribution` as well? We'll need to migrate that too?

Reply:
Yeah, it looks like that was missed here.